js/src/components/paid-ads/campaign-assets-form.js

```diff
@@ -1,28 +1,83 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
 import AdaptiveForm from '.~/components/adaptive-form';
 import validateCampaign from '.~/components/paid-ads/validateCampaign';
+import validateAssetGroup from '.~/components/paid-ads/validateAssetGroup';
 
 /**
  * @typedef {import('.~/components/types.js').CampaignFormValues} CampaignFormValues
+ * @typedef {import('.~/components/types.js').AssetGroupFormValues} AssetGroupFormValues
  */
 
+const emptyAssetGroup = {
```

Comment:
💅  Maybe we can rename this in uppercase `EMPTY_ASSET_GROUP`

Reply:
Unless variables will be exposed outside a module, it does not seem to be a clear convention for variables used only inside a module. Searching this codebase globally with `^const [a-z]\w* = [\[\{]` shows plenty of non-all-uppercase naming.